components/mediation/data-publishers/org.wso2.micro.integrator.prometheus.publisher/pom.xml

```diff
@@ -69,5 +69,13 @@
             <artifactId>org.apache.felix.scr.ds-annotations</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>io.prometheus</groupId>
+            <artifactId>simpleclient</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.prometheus</groupId>
+            <artifactId>simpleclient_common</artifactId>
```

Comment:
How are we packing these or are these coming from somewhere else already?

Reply:
Done in https://github.com/wso2/micro-integrator/pull/1556/